### ISSUE TITLE
:math: not working on runestone.academy

### ISSUE THREAD

**Issue by irunestone** [RunestoneInteractive/RunestoneComponents#872]
Error reported in course test2 on page N/A by user tluchko tluchko@csun.edu
When trying to use the rst :math: directive in a new exercise in my course, I simply get the raw text surrounded by '\(' and '\)'.  However, if I run the server locally on my Mac, following the directions at
 https://runestone.academy/runestone/static/instructorguide/WritingExercises/advancedRunestone.html

the same directive renders the math correctly.

Below is a minimum working example.  The math in the example renders as '\(a^2\)' using the Runestone website but works as expected on my local server.

.. activecode:: math_does_not_work
   :language: python
  
   Simple test of the math directive: :math:`a^2`
   ~~~~

   print("hello world")

   ====

   print("Hidden code, such as unit tests come after the four = signs")

**Comment by bnmnetp** [comment]
There are two problems:

1.  mathjax was not being loaded for this page -- fixed.
2. Even when it is loaded, we need to find a way to tell mathjax to do its work when we dynamically load in the preview.

**Comment by bjones1** [comment]
Yep -- we should follow the instructions at https://docs.mathjax.org/en/v1.0/typeset.html in two cases:

1. When previewing a problem.
2. When revealing answers -- if there's math in the feedback for a solution, it must be rendered.

**Comment by bnmnetp** [comment]
fixed by commit #8261484
